app/pages/home-not-logged-in/research.jsx

```diff
@@ -43,7 +39,7 @@ counterpart.registerTranslations('en', {
 const HomePageResearch = (({ count, screenWidth, showDialog, volunteerCount }) =>
   <section className="home-research">
     <Translate className="tertiary-kicker" component="h2" content="researchHomePage.works" />
-    <span className="class-counter">{count.toLocaleString()}</span>
+    <span className="class-counter">{count.toString().replace(/.(?=(?:.{3})+$)/g, '$&,')}</span>
```

Comment:
Yeah, originally this issue came up because it was reported `.toLocaleString()` wasn't working on a Safari browser, and I'm suspecting it was an older version of the browser. The `.toLocaleString()` function appears to work on current Safari. 

This brings up a question about browser support. Should we make concessions to support older versions of a browser or expect browsers to be up to date?